.github/workflows/build-test-macos.yml

```diff
@@ -26,7 +26,7 @@ concurrency:
 
 jobs:
   macos-unittest:
-    runs-on: macos-latest
+    runs-on: macos-11
```

Comment:
We can build on older and use on newer, but not the other way around. This is the oldest supported macos version by gha